### DISCUSSION TITLE
UQL CSV where operator

### DISCUSSION THREAD

**Discussion by greenmotion** [grafana/grafana-infinity-datasource#419] [Q&A] [answered]
Currently running infinity-datasource 1.0.0 (great plugin by the way, well done!), but I can't seem to get the "where" operator to work in UQL CSV.

When I do a:

```
parse-csv
| project "Opportunity Financial Year", "Customer Name"
```

I get a result set like:

```
Opportunity Financial Year	Customer Name
FY2022-Q1			Customer-A
FY2022-Q2			Customer-B
FY2022-Q3			Customer-C
FY2022-Q4			Customer-D
```

But when I change it to:

```
parse-csv
| project "Opportunity Financial Year", "Customer Name"
| where "Opportunity Financial Year" = 'FY2022-Q1'
```

I get "No data" in my table view.

What am I doing wrong?

Thank you.

**Comment by yesoreyeram** [comment, marked as answer]
It is `==` instead `=`

```csv
Opportunity Financial Year,Customer Name
FY2022-Q1,Customer-A
FY2022-Q2,Customer-B
FY2022-Q3,Customer-C
FY2022-Q4,Customer-D
```

```kusto
parse-csv
| project "Opportunity Financial Year", "Customer Name"
| where "Opportunity Financial Year" == 'FY2022-Q1'
```

<img width="1293" alt="image" src="https://user-images.githubusercontent.com/153843/193591840-20559205-cf89-427a-91bb-959c961ae743.png">
